### PR TITLE
test(bulletin-board):[bulletin-board] update bulletin-board E2E test

### PR DESCRIPTION
**Pull request by wuyiping0628**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/sites/demos/pc/app/bulletin-board/url.spec.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ test('BulletinBoard 基于Url的跳转(', async ({ page }) => {
   page.on('pageerror', (exception) => expect(exception).toBeNull())
```
Comment (shenjunjian) [inline examples/sites/demos/pc/app/bulletin-board/url.spec.ts:4]: 标题多了一个 (

File: examples/sites/demos/pc/app/bulletin-board/url.spec.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ test('BulletinBoard 基于Url的跳转(', async ({ page }) => {
...
   await page.goto('bulletin-board#url')
 
-  const preview = page.locator('#preview')
+  const preview = page.locator('#url')
   const contentTitle = preview.locator('.tiny-bulletin-board__textTitle')
 
   // a标签href属性值
```
Comment (shenjunjian) [inline examples/sites/demos/pc/app/bulletin-board/url.spec.ts:10]: 这里推荐使用 await  expact (locator).toXXX 的模式。

File: examples/sites/demos/pc/app/bulletin-board/url.spec.ts (hunk 1 above)
Comment (wuyiping0628) [reply examples/sites/demos/pc/app/bulletin-board/url.spec.ts:4]: 已修改

File: examples/sites/demos/pc/app/bulletin-board/url.spec.ts (hunk 2 above)
Comment (wuyiping0628) [reply examples/sites/demos/pc/app/bulletin-board/url.spec.ts:10]: 已修改

**Review by zzcr — APPROVED** [review]

**Pull request merged**